lib/src/Pos/Block/Core/Union/Types.hs

```diff
@@ -30,39 +28,21 @@ import           Pos.Core.Block
 ----------------------------------------------------------------------------
 
 -- | Ssc w/ buildable blockchain
-type BiSsc ssc =
-    ( Ssc ssc
-    , Bi (GenericBlockHeader (GenesisBlockchain ssc))
-    , Bi (GenericBlockHeader (MainBlockchain ssc))
+type BiSsc =
```

Comment:
This name looks even more weird now (as well as the comment above).

Reply:
Sure, I'm going to remove it.